INTRODUCTION.md

```diff
@@ -241,7 +241,7 @@ on the producer:
 
 See [STATISTICS.md](STATISTICS.md) for the full definition of metrics.
 A JSON schema for the statistics is available in
-[statistics-schema.json](src/statistics-schema.json).
+[statistics_schema.json](https://github.com/edenhill/librdkafka/blob/master/src/statistics_schema.json).
```

Comment:
I think we can still use relative links here. The broken link was the result of using `-` instead of `_`.

I would update this to 
```
[statistics-schema.json](src/statistics_schema.json).
```